dart/tool/service_undocumented.md

```diff
@@ -158,3 +158,53 @@ class HeapSpace extends Response {
 <!-- _getCpuProfileTimeline -->
 
 <!-- _getAllocationSamples -->
+
+streamId | event types provided
```

Comment:
I think that the change will make things easier in the future even if we make the API public.

Right now the __Dart__ version supports extra stream and events that the __Java__ version does not, due to the fact that the parse and code generator does not support that and they are manually added before code generation.